ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift

```diff
@@ -58,6 +58,16 @@ public class UserCollection: ObservableObject {
         return caught
     }
     
+    public func collectedIn(list: [Item]) -> Int {
```

Comment:
You can reuse the `caughtIn` function, as it's an array of item, just filter out any "fake" art, you'll see in the next comment I recommend to check if the name contain fake instead of using the price.